camp/nfutsum/nfutsum-miniproject/XBox Controller Left and Right bumper/src/main/java/frc/robot/commands/Forward.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.Motor;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+/** An example command that uses an example subsystem. */
+public class Forward extends CommandBase {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
```

Comment:
should not need to suppress warnings here. should fix the problems causing the warnings.

Reply:
removed